aya-build/src/lib.rs

```diff
@@ -147,3 +147,21 @@ pub fn build_ebpf(packages: impl IntoIterator<Item = Package>) -> Result<()> {
     }
     Ok(())
 }
+
+#[derive(Default)]
+pub enum Toolchain<'a> {
+    /// Use the latest nightly toolchain to compile to eBPF kernel, e.g. `+nightly`.
+    #[default]
+    Nightly,
+    /// Uses a custom toolchain to compile to eBPF kernel, e.g. `+nightly-2021-01-01`.
```

Comment:
please clarify whether the leading plus is required or not. my preference is "not"